TU-Agent/tygron.mod2g

```diff
@@ -39,8 +39,18 @@ module tygron {
 	
 	%if the buildConstruction module adopted the goal to create land we should enter the demolish module
 	%TODO: We could also buy land to create buildable land
-	if goal(createLandToBuild)
+	if goal(createLandToBuild), bel(oldBuildings([Head|Tail]))
 		then demolish.
+		
+	%If we don't have more buildings to destroy, we need to buy some land so we enter the buyLand module
+	if goal(createLandToBuild)
+		then buyLand.
+	
+	%Calls the upgrade module
+	%Only does this when all high buildings have been built and we don't need more trees for our indicator
+	if bel(upgrades(_), not(upgrades([]))), goal(indicatorGoal('Bouw onderwijs', _)), 
+		not(goal(indicatorGoal('Parken en groen', _))), not(goal(indicatorgoal('Bouwen hoge gebouwen', _)))
```

Comment:
Yes, codecleanup fixes this

Sent from my iPhone

> On 17 Jun 2016, at 10:28, "MichielvdBerg" <notifications@github.com> wrote:
> 
> In TU-Agent/tygron.mod2g:
> 
> >  		then demolish.
> > +		
> > +	%If we don't have more buildings to destroy, we need to buy some land so we enter the buyLand module
> > +	if goal(createLandToBuild)
> > +		then buyLand.
> > +	
> > +	%Calls the upgrade module
> > +	%Only does this when all high buildings have been built and we don't need more trees for our indicator
> > +	if bel(upgrades(_), not(upgrades([]))), goal(indicatorGoal('Bouw onderwijs', _)), 
> > +		not(goal(indicatorGoal('Parken en groen', _))), not(goal(indicatorgoal('Bouwen hoge gebouwen', _)))
> Indicatorgoal need to be indicatorGoal right?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>